docs/references/sdk/frontend-only.mdx

```diff
@@ -1,7 +1,152 @@
 ---
-description: TODO
+description: A reference for implementing a frontend-only Clerk SDK
 ---
 
 # Frontend-only SDK
```

Comment:
The purpose of this doc is to give users guidance on how to create a Clerk SDK for a frontend-only framework. It's assumed that they've read the other pages including the terminology page